lib/init.c

```diff
@@ -80,6 +85,7 @@ static int probe_sequence[] = {
   PCI_ACCESS_OBSD_DEVICE,
   PCI_ACCESS_DARWIN,
   PCI_ACCESS_SYLIXOS_DEVICE,
+  PCI_ACCESS_GNU,
```

Comment:
Should we call this GNU if the back-end calls itself "hurd"?

Reply:
Done in 1cedbcc